CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# 2.1.1
```

Comment:
Minor detail I forgot to mention: change to Unreleased as described here https://github.com/folkehelseinstituttet/Fhi.Frontend.Style/blob/main/CONTRIBUTING.md#feature-branches (Then you can make a release branch as next step, or wait for more features before makin a new release...)